site/components/layout/HomeFooter.js

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import { useTranslations } from 'next-intl'
+
+import SvgContainer from '../svg/SvgContainer'
+
+const HomeFooter = function () {
+  const t = useTranslations()
+  return (
+    <div className="border-slate-300/55 flex items-center justify-between mt-8 p-8 text-gray-500 border-t">
+      <div>
+        <p>{t('copyright')}</p>
+      </div>
+      <div className="flex gap-3">
+        <Link href="https://x.com/hemi_xyz" target="_blank">
```

Comment:
Next'js `Link` should be used for relative links. For external links (As are all on this page), an anchor tag (`a`) should be used.

It is also a good practice to add `rel="noopener noreferrer"`, though depending on the source you check, one of both may be redundant (or maybe one is not recognized by certain browsers). So playing safe I'd just add both

Reply:
Fixed